Formula/t/temporal.rb

```diff
@@ -21,7 +21,7 @@ class Temporal < Formula
   def install
     ldflags = "-s -w -X github.com/temporalio/cli/headers.Version=#{version}"
     system "go", "build", *std_go_args(ldflags:), "./cmd/temporal"
-    generate_completions_from_executable(bin/"temporal", "completion", shells: [:bash, :zsh])
+    generate_completions_from_executable(bin/"temporal", "completion", shells: [:bash, :fish, :zsh])
```

Comment:
you can just remove the shells block now (by default we support all three shells)
```suggestion
    generate_completions_from_executable(bin/"temporal", "completion"
```